src/xdp/program.c

```diff
@@ -1635,7 +1635,7 @@ XdpProgramCanXskBypass(
         Program->Rules[0].Match == XDP_MATCH_ALL &&
         Program->Rules[0].Action == XDP_PROGRAM_ACTION_REDIRECT &&
         Program->Rules[0].Redirect.TargetType == XDP_REDIRECT_TARGET_TYPE_XSK &&
-        XskIsDatapathHandleQueueMatched(Program->Rules[0].Redirect.Target, RxQueue);
+        XskCanBypass(Program->Rules[0].Redirect.Target, RxQueue);
```

Comment:
nit: inline XskCanBypass?

Reply:
The XSK struct is defined in the xsk.c file, so it's hard to inline without exposing XSKs to everyone.

Reply:
Unless you mean just provide a hint to the compiler?

Reply:
I'm going to merge this because my own testing is dependent on it, but I'm happy to follow up.